regulations/static/regulations/js/source/collections/comment-collection.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var $ = require('jquery');
+var Backbone = require('backbone');
+Backbone.LocalStorage = require('backbone.localstorage');
+
+var CommentModel = require('../models/comment-model');
+
+var CommentCollection = Backbone.Collection.extend({
+  model: CommentModel,
+  localStorage: new Backbone.LocalStorage('comment:' + window.DOC_NUMBER)
```

Comment:
I was thinking of a `CommentCollection` as a set of comments targeting a single proposal. If a user switches to comment on a different proposal, they'll get a new collection. If we stored comments for all proposals in the same collection, we'd always have to filter by proposal--not that that's hard, but I'm not (yet) seeing why it's better.